src/app/shared/form/builder/ds-dynamic-form-ui/ds-dynamic-form-control-container.component.ts

```diff
@@ -212,4 +309,38 @@ export class DsDynamicFormControlContainerComponent extends DynamicFormControlCo
       this.onChange(event);
     }
   }
+
+  public hasResultsSelected(): Observable<boolean> {
+    return this.model.value.pipe(map((list: Array<SearchResult<DSpaceObject>>) => isNotEmpty(list)));
+  }
+
+  openLookup() {
```

Comment:
Again a brief description or TypeDocs.   Also see `removeSelection` just below this.